packages/web3/std/nft_collection_interface.ral

```diff
@@ -26,4 +26,6 @@ Interface INFTCollection {
    pub fn totalSupply() -> U256
 
    pub fn nftByIndex(index: U256) -> INFT
+
+   pub fn validateNFT(nftId: ByteVec, nftIndex: U256) -> ()
```

Comment:
Let's add a comment that this should throw an error if the validation failed.

Reply:
Sounds great :+1: